.github/workflows/build_tests.yaml

```diff
@@ -88,6 +88,10 @@ jobs:
       run: python3 xpk.py workload delete --workload $WORKLOAD_NAME --cluster $TPU_CLUSTER_NAME --zone=us-central2-b
     - name: Delete the Pathways workload on the cluster
       run: python3 xpk.py workload delete --workload $PATHWAYS_WORKLOAD_NAME --cluster $TPU_CLUSTER_NAME --zone=us-central2-b
+    - name: Run a batch job on the cluster
+      run: python3 xpk.py batch --cluster $TPU_CLUSTER_NAME --zone=us-central2-b test.sh
+    - name: List out the jobs on the cluster
+      run: python3 xpk.py job ls | grep 'xpk-def-app-profile-slurm-'
```

Comment:
I think we should fail test if something not correct.

Reply:
As I understand, grep will exit with code 1 if the pattern `xpk-def-app-profile-slurm-'` is not found which will make the test failing

Reply:
Ah I see. Thanks!